src/main/java/com/keronic/Majestik.java

```diff
@@ -3,57 +3,48 @@
 
 import module java.base;
 
-import com.keronic.antlr4.MajestikBaseVisitor;
-import com.keronic.antlr4.MajestikLexer;
-import com.keronic.antlr4.MajestikParser;
-import com.keronic.antlr4.MajestikParser.ProgContext;
 import com.keronic.majestik.MajestikRuntimeException;
+import com.keronic.majestik.ast.MajestikCodeVisitor;
 import java.lang.System.Logger.Level;
-import org.antlr.v4.runtime.CharStream;
-import org.antlr.v4.runtime.CharStreams;
-import org.antlr.v4.runtime.CommonTokenStream;
+import nl.ramsolutions.sw.MagikToolsProperties;
+import nl.ramsolutions.sw.magik.MagikFile;
```

Comment:
Looks fine to me, can you elaborate?

Reply:
`MethodHandles` should not be imported since it is imported by the module import. The imports looks sorted to me, can you point me to the actual issue?